interfaces/SimpleDVTStakingStrategy.json

```diff
@@ -0,0 +1,619 @@
+[
```

Comment:
let's squeeze this also, and other abis)

Reply:
Done